Cargo.toml

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: GPL-2.0
+
+# TODO: generate automatically
+[workspace]
+members = [
+    "rust/shlex",
```

Comment:
I don't think so. Other third-party code in the kernel is included as-is.

That module is likely to go away anyway, it is just an example of how things will look like for intra-kernel dependencies. The reason is that we are requiring to fetch other dependencies from the network at the moment, so it is not helpful to avoid just one of them.